homeassistant/components/honeywell/__init__.py

```diff
@@ -2,7 +2,7 @@
 import asyncio
 from dataclasses import dataclass
 
-import AIOSomecomfort
+import aiosomecomfort as AIOSomecomfort
```

Comment:
you haven't tested this.

Reply:
I did test it in the dev environment and it is working,  It was defiantly weird as PyPi installed it just fine (and it worked), but someone tried to install from the repo and it failed (due to case sensitivity).  
Should I do something more to the API or just change the HA code (removing the `as` part)?

Reply:
try it 👍

Reply:
Rather than being lazy I changed all of them to lowercase :)